Content.Server/Devour/DevourSystem.cs

```diff
@@ -37,6 +38,10 @@ private void OnDoAfter(EntityUid uid, DevourerComponent component, DevourDoAfter
             }
             _bloodstreamSystem.TryAddToChemicals(uid, ichorInjection);
         }
+        else if (component.ShouldStoreDevoured && HasComp<BorgChassisComponent>(args.Args.Target)) 
+        {
+              ContainerSystem.Insert(args.Args.Target.Value, component.Stomach);
+        }
```

Comment:
1. Why is there 2 checks for container insertion.
2. Don't hardcode in components, at best use a whitelist.

Reply:
Never used a whitelist before.
How would I do that?

Reply:
Take a look at #29332!